src/index.js

```diff
@@ -412,7 +412,7 @@ export function check(pretty = true) {
 		}
 
 		// Check if default is set
-		if (opts.default) {
+		if (opts.default != null) {
```

Comment:
Unless I misread something, using `in` would always be true since `defaultOpts` sets `default` to `null` when missing, so the object always has the property.